src/wp-admin/includes/update-core.php

```diff
@@ -1672,6 +1672,17 @@ function _upgrade_440_force_deactivate_incompatible_plugins() {
  */
 function _upgrade_580_force_deactivate_incompatible_plugins() {
 	if ( defined( 'GUTENBERG_VERSION' ) && version_compare( GUTENBERG_VERSION, '10.7', '<=' ) ) {
+		$deactivated_plugins = get_option( 'wp_deactivated_plugins', array() );
+		$deactivated_plugins['gutenberg'] = array(
+			'plugin_name'         => 'Gutenberg',
+			'version_deactivated' => GUTENBERG_VERSION,
+			'version_compatible'  => '10.8',
+		);
+		if ( is_plugin_active_for_network( 'gutenberg/gutenberg.php' ) ) {
```

Comment:
Would the MU plugin scenario we encountered in [Core-53410](https://core.trac.wordpress.org/ticket/53410) come into play here?

Reply:
> Would the MU plugin scenario we encountered in [Core-53410](https://core.trac.wordpress.org/ticket/53410) come into play here?

Unfortunately no, sites with an old version of GB as an mu-plugin will fatal until the developer updates and there isn't a way to protect against that. I'm not sure it's even possible to detect if it's active as an mu-plugin.

Reply:
> I'm not sure it's even possible to detect if it's active as an mu-plugin

A check to see if `GUTENBERG_VERSION` is defined AND Gutenberg isn't in ` wp_get_active_network_plugins` or `wp_get_active_and_valid_plugins`, that would mean that it is either being loaded from a theme OR is an MU plugin.